src/WasBornAfterTest.java

```diff
@@ -78,17 +75,43 @@ void should_ReturnTrue_When_PersonDateOfBirth_IsEqualTo() {
     LocalDate PersonD = LocalDate.parse("1978-01-12");
     LocalDate PersonE = LocalDate.parse("1996-08-20");
 
-
     //then (assert that results are as expected)
-    assertThat(personArray).contains(a,b,c,d,e);
+    assertThat(personArray).contains(a, b, c, d, e);
 
     assertThat(personArray[0].getDateOfBirth()).isEqualTo(PersonA);
     assertThat(personArray[1].getDateOfBirth()).isEqualTo(PersonB);
     assertThat(personArray[2].getDateOfBirth()).isEqualTo(PersonC);
     assertThat(personArray[3].getDateOfBirth()).isEqualTo(PersonD);
     assertThat(personArray[4].getDateOfBirth()).isEqualTo(PersonE);
-  }}
+  }
 
+  @Test
+  void shouldReturnTrue_When_SearchAndPrintPerson(){
+    //given
+    List<Person> people = new ArrayList<>();
+    people.add(new Person("John", LocalDate.parse("1969-03-15"), "+447984356766", "john@example.com"));
+    people.add(new Person("Jane", LocalDate.parse("1998-04-09"), "+447220512328", "jane@example.com"));
+    people.add(new Person("Harry",LocalDate.parse("1980-09-25"), "+447220012555", "harry@example.com"));
+    people.add(new Person("Anne", LocalDate.parse("1978-01-12"), "+447220012222", "anne@example.com"));
+    people.add(new Person("Jack", LocalDate.parse("1996-08-20"), "+447220012098", "jack@example.com"));
+    //when, then
 
+    assertTrue(SortPeopleByDateOfBirth.searchAndPrintPerson(people, "jane@example.com", "+447220512328"));
 
+  }
 
+  @Test
+  void shouldReturnFalse_When_SearchAndPrintPerson(){
+    //given
+    List<Person> people = new ArrayList<>();
+    people.add(new Person("John", LocalDate.parse("1969-03-15"), "+447984356766", "john@example.com"));
+    people.add(new Person("Jane", LocalDate.parse("1998-04-09"), "+447220512328", "jane@example.com"));
+    people.add(new Person("Harry",LocalDate.parse("1980-09-25"), "+447220012555", "harry@example.com"));
+    people.add(new Person("Anne", LocalDate.parse("1978-01-12"), "+447220012222", "anne@example.com"));
+    people.add(new Person("Jack", LocalDate.parse("1996-08-20"), "+447220012098", "jack@example.com"));
+    //when, then
+
+    assertFalse(SortPeopleByDateOfBirth.searchAndPrintPerson(people, "joe@example.com", "+447720512328"));
+
+  }
```

Comment:
I think we could add a few more test case scenarios for a matching phone number but not email. The only part of your conditional that is being tested currently is only if the email matches but we're not verifying the same is true for phone numbers :)